pkg/store/store.go

```diff
@@ -38,6 +38,7 @@ type Store interface {
 	Update(ctx context.Context, key Key, updater func(*unstructured.Unstructured) error) error
 	IsLoading(ctx context.Context, key Key) bool
 	Create(ctx context.Context, object *unstructured.Unstructured) error
+	CreateFromYAML(ctx context.Context, namespace, input string) ([]string, error)
```

Comment:
"Create" is only half of what this call actually does.

```suggestion
	ApplyFromYAML(ctx context.Context, namespace, input string) ([]string, error)
```

or 

```suggestion
	CreateOrUpdateFromYAML(ctx context.Context, namespace, input string) ([]string, error)
```

Reply:
Yep, this is better. I'll change it.